DependencyInjection/Configuration.php

```diff
@@ -112,6 +112,14 @@ private function getMailersNode()
                 ->end()
                 ->scalarNode('sender_address')->end()
                 ->scalarNode('delivery_address')->end()
```

Comment:
and then you should check below if the passed value isn't an array (and cast it then)